x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/logs_ui_categories/ml/log_entry_categories_count.json

```diff
@@ -0,0 +1,34 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "Categorize log entries and detect count anomalies",
```

Comment:
For the SIEM modules, we've started the job description with a reference to the module. How about something like 'Log entry categories: Looks for anomalies in counts', or 'Logs UI: Looks for anomalies in count by log entry category'?